<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>schemaDialog</class>
 <widget class="QDialog" name="schemaDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>580</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>580</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Schema Editor</string>
  </property>
  <widget class="QLabel" name="textLabel1_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>100</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text" >
    <string>8-15</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clr1Button" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>60</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clr2Button" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>60</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clr3Button" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>60</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clr4Button" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>60</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clr5Button" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>60</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clr6Button" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>60</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clr7Button" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>60</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="titleLineEdit" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>20</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clr0Button" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>60</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clr9Button" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>90</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clr10Button" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>90</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clr11Button" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>90</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clr12Button" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>90</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clr13Button" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>90</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clr14Button" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>90</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clr15Button" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>90</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clr8Button" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>90</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>70</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>0-7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>340</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>340</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Remove </string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton" >
   <property name="geometry" >
    <rect>
     <x>490</x>
     <y>400</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>400</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>310</x>
     <y>9</y>
     <width>261</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title" >
    <string>Background</string>
   </property>
   <widget class="QLabel" name="TextLabel1" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>300</y>
      <width>66</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Fade to</string>
    </property>
   </widget>
   <widget class="QLabel" name="TextLabel2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>330</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Min</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="noneRadioButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>None</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imageLineEdit" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>70</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseButton" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>70</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>Choose</string>
    </property>
   </widget>
   <widget class="QLabel" name="imagePixmapLabel" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>110</y>
      <width>170</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="scaledContents" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="alphaSlider" >
    <property name="geometry" >
     <rect>
      <x>39</x>
      <y>330</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="TextLabel3" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>330</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Max</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fadeButton" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>300</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="typeComboBox" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>40</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>Tile</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Center</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Scale</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="imageRadioButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Image</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="transpRadioButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>270</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Transparent</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="nameListWidget" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>130</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
